src/Symfony/Component/AssetMapper/Compiler/JavaScriptImportPathCompiler.php

```diff
@@ -50,7 +50,7 @@ final class JavaScriptImportPathCompiler implements AssetCompilerInterface
             )
             \s*[\'"`](\.\/[^\'"`\n]++|(\.\.\/)*+[^\'"`\n]++)[\'"`]\s*[;\)]
         ?
-    /mx';
+    /mxu';
```

Comment:
This will break parsing non UTF8 files. My previous suggestion was made to handle such cases.
We can merge as is and wait for someone to fill in a bug, maybe UTF-8 is the only encoding found in practice?
:shrug: